### PR TITLE
CLDR-15080 Fix broken links for Survey Tool instructions at sites.google.com

### PR DESCRIPTION
**Pull request by btangmu**
-Change some links from sites.google.com to cldr.unicode.org

-TODO comment where I could not find new page

-Remove defunct bugFeedbackUrl for legacy st_footer.jsp

CLDR-15080

- [ ] This PR completes the ticket.

### CODE REVIEW

File: tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckQuotes.java [hunk 1]
```diff
@@ -10,6 +10,8 @@
 import com.ibm.icu.text.UnicodeSet;
 
 public class CheckQuotes extends CheckCLDR {
+    private static final String VALID_DELIMITER_URL = "https://cldr.unicode.org/translation/characters";
```
Comment (AEApple) [inline tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckQuotes.java:13]: So this page was originally at: https://cldr.unicode.org/translation/characters-emoji-symbols 

And had 3 subpages:

https://cldr.unicode.org/translation/characters/character-labels
https://cldr.unicode.org/translation/characters/emoji-names-and-keywords -> parameter would need to be updated to align with original.
https://cldr.unicode.org/translation/characters/typographic-names

I am happy to revert to match the old site, but I'd like to confirm that you agree on the approach before making the changes.

Comment (AEApple) [reply tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckQuotes.java:13]: Will update the site to link the subpages on the main header page which didn't migrate properly.

**Review by AEApple — APPROVED** [review]

File: tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckQuotes.java (hunk 1 above)
Comment (btangmu) [reply tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckQuotes.java:13]: Per today's meeting I'll change all remaining "characters-emoji-symbols" to "characters"

Comment (AEApple) [reply tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckQuotes.java:13]: Actually do you think this was the original characters page? https://cldr.unicode.org/translation/core-data/alphabetic-information

I am wondering after looking up some of the other links you weren't able to find originally.

**Pull request merged**